composer.json

```diff
@@ -21,12 +21,15 @@
   "require": {
     "php": ">=5.6.0",
     "symfony/yaml": "^3.0",
-    "composer/installers": "~1.0"
+    "composer/installers": "~1.0",
+    "ext-zip": "^1.15",
+    "phpmailer/phpmailer": "^6.1"
   },
   "autoload": {
     "psr-4": {
-      "Drush\\Commands\\acsf_tools\\": "."
-    }
+        "Drush\\Commands\\acsf_tools\\": "src"
```

Comment:
Why such change?

Reply:
@alex-moreno please help here

Reply:
it is a better practice to keep all classes in the src folder following psr4 standards. To ensure it's not breaking the current acsf objects I map manually as well the current folder (next line)